src/utils/__tests__/changelog.test.ts

```diff
@@ -487,8 +487,7 @@ describe('generateChangesetFromGit', () => {
         },
         M5: {
           title: 'Better driver experience',
-          description:
-            'We are working on making your driving experience more pleasant and safer.',
+          description: '',
```

Comment:
Even more of an edge case (and not necessary for this PR) ... what if the description is `'  '` ... i.e., non-empty whitespace?

Reply:
We can trim it but I'd assume GitHub would do that already.